app/src/main/java/com/example/parkingsystemjava/mvp/view/DialogFragmentView.java

```diff
@@ -0,0 +1,25 @@
+package com.example.parkingsystemjava.mvp.view;
+
+import com.example.parkingsystemjava.DialogFragmentParkingLots;
+import com.example.parkingsystemjava.databinding.DialogFragmentParkingLotsBinding;
+import com.example.parkingsystemjava.mvp.contract.DialogFragmentContract;
+import java.lang.ref.WeakReference;
+
+public class DialogFragmentView implements DialogFragmentContract.ViewContract {
+    private DialogFragmentParkingLotsBinding binding;
+    private WeakReference<DialogFragmentParkingLots> dialogFragmentRef;
+
+    public DialogFragmentView(DialogFragmentParkingLotsBinding binding, DialogFragmentParkingLots dialogFragmentParkingLots) {
+        this.binding = binding;
+        this.dialogFragmentRef = new WeakReference<>(dialogFragmentParkingLots);
```

Comment:
please create a FragmentView in the base package, just like the ActivityView, to put all the generic things like the weak reference

Reply:
done